src/main/java/br/com/autonomiccs/apacheCloudStack/client/ApacheCloudStackClient.java

```diff
@@ -192,15 +193,23 @@ protected HttpContext createHttpContextWithAuthenticatedSessionUsingUserCredenti
     }
 
     /**
+     * It createsa {@link RequestConfig} object with configured {@link #connectionTimeout} value.
+     * This can be used to set on a HttpClient its connect timeout, connection request timeout and socket timeout.
+     * @return
+     */
+    protected RequestConfig createRequestConfig() {
+        return RequestConfig.custom()
```

Comment:
Thanks @rafaelweingartner I've incorporated your feedback, please re-review the changes now.